docs/index.md

```diff
@@ -1642,12 +139,34 @@ system is the latest Ubuntu LTS the provider supports.
 | Fedora                     | 21+              | experimental            |
 
 To use a different base operating system on a remote provider, specify the
-provider's image flag and one of its available images. For example, to
-select a `debian-8-x64` image on DigitalOcean you would supply the following:
-
-    --digitalocean-image=debian-8-x64
+provider's image flag and one of its available images. For example, to select a
+`debian-8-x64` image on DigitalOcean you would supply the
+`--digitalocean-image=debian-8-x64` flag.
 
 If you change the base image for a provider, you may also need to change
 the SSH user. For example, the default Red Hat AMI on EC2 expects the
 SSH user to be `ec2-user`, so you would have to specify this with
 `--amazonec2-ssh-user ec2-user`.
+
+## Getting help
```

Comment:
This is a non-standard presentation for this type of material.  I am making this consistent to what we do elsewhere in the documentation.